cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,86 @@
+The tool provisions the infrastructure for a CloudHub 2.0 private space by cloning the existing configurations of your eligible CloudHub dedicated VPC, using the same CIDR block. If your VPC has existing TGW or VPN network connections, they are seamlessly transferred to the new CloudHub 2.0 private space. This ensures that any new applications deployed into the new private space can communicate effectively.
+
+== Understand Eligibility
+//Which VPCs are eligible for upgrade
+
+Due to their architectural differences and distinct interfaces, CloudHub 2.0 doesn’t support some features in the same way CloudHub does. Follow this eligibility check to confirm which VPCs are eligible for upgrade:
+
+Availability of IPs::
++
+Your VPC must have at least 25% free space, and 100 or more available IP addresses per subnet to be eligible for upgrade.
+
+Deprecated Availability Zones (AZs)::
++
+VPCs using deprecated AZs aren’t eligible for the upgrade.
+
+Legacy VPNs::
++
+VPCs with legacy VPNs configured aren’t eligible for the upgrade.
+
+Direct Connect or VPC Peering:: 
+If your VPC has Direct Connect or VPC Peering configured in their VPC, migrate to Transit Gateway or VPN before upgrading.
+
+TLS 1.1 ciphers::
++
+These ciphers can’t be carried over from CloudHub to CloudHub 2.0:
++
+* ECDHE-ECDSA-AES128-SHA1
+* ECDHE-ECDSA-AES256-SHA1
+* ECDHE-RSA-AES128-SHA1
+* ECDHE-RSA-AES256-SHA1
+
+
+== Upgrade your VPC via Anypoint Platform
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Upgrade*.
++
+Alternatively, select *Runtime Manager* > *What's New* to read about how the VPC upgrade works, and click *Continue to Upgrade* to start upgrading your VPCs.
+. Click *Upgrade Now*, under the *Upgrade Status* column for the VPC you want to upgrade.
++
+The *Upgrade to CloudHub 2.0* review page displays the configurations currently available in your VPC, as well as the configurations that are created during the upgrade.
++
+image::upgrade-ch2-review-conf.png[]
++
+. Click *Next* to continue.
+. Provide a name for the new private space.
++ 
+Optionally, you can specify xref:ps-gather-setup-info.adoc#cidr-block[reserved CIDRs] to connect to your new private space.
+. Click *Create* to create the new private space.
++
+The *Upgrade to CloudHub 2.0* review page shows the progress of the VPC upgrade process, which takes approximately fifteen to thirty minutes.
+
+As soon as the upgrade process starts, the upgraded VPC is locked for any further updates. 
+
+You can cancel the upgrade process to revert to your VPC at any time by clicking *Cancel Upgrade*. In that case, the newly created private space is deleted, and your VPC is unlocked.
```

Comment:
Once the private space is successfully provisioned, the 'Cancel Upgrade' button is hidden. However, the user can still rollback by deleting the private space. We need to add this as well.